app/scripts/modules/kubernetes/serverGroup/configure/wizard/advancedSettings.html

```diff
@@ -1,5 +1,31 @@
 <ng-form name="advancedSettings">
   <div class="form-group form-horizontal">
+
+    <div class="form-group">
```

Comment:
Super minor, but it feels as if the ordering here should match the ordering in the details panel.